app/pulp/app/tasks/publisher.py

```diff
@@ -1,8 +1,12 @@
 from celery import shared_task
 
+from logging import getLogger
+
 from pulp.app import models
 from pulp.tasking.tasks import UserFacingTask
 
+_logger = getLogger(__name__)
```

Comment:
Can this be moved into a page in the docs called logging.rst? I would put it here https://github.com/pulp/pulp/tree/3.0-dev/docs/contributing and it would provide a basic logging example.